src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -22,7 +22,7 @@
 
 @Service
 @ConditionalOnProperty(prefix = "core_case_data", name = "api.url")
-public class CCDClaimSearchRepository {
+public class CCDCaseApi {
```

Comment:
The name is a bit unfortunate but I am happy o accept it for now as in the future the implementation is likely to be moved to `CCDCaseRepository`.